cves/CVE-2012-3442.yml

```diff
@@ -109,10 +119,13 @@ unit_tested:
 
     For the fix_answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: false
+  code_answer: |
+    Author does not disclose how they discovered the vulnerability.
+  fix: true
+  fix_answer: |
+    A unit test called "test_unsafe_redirect" was added to 
+    `tests/regressiontests/httpwrappers/tests.py`
```

Comment:
I just wanted to say thank you for being the only other person I've seen to mention what unit test(s) were added.